docs/run-a-node/setup/install.md

```diff
@@ -263,13 +268,19 @@ https://algorand-catchpoints.s3.us-east-2.amazonaws.com/channel/mainnet/latest.c
 The results will look similar to this:
 `4420000#Q7T2RRTDIRTYESIXKAAFJYFQWG4A3WRA3JIUZVCJ3F4AQ2G2HZRA`
 
+!!! Note 
+    It has been noted that in some instances the `datafastcatchup` is not found. Resulting in a 
+    Algorand node failed to start: exit status 1. Please create a directory named `datafastcatchup` and copy the contents of the `data` folder into it.
+
```

Comment:
I think that adding the `datafastcatchup` directory should not be part of this manual. It would only make it more confusing.

Reply:
Agreed, I think it would be a good idea to focus on Sandbox. If you are not willing to fix it.